### DISCUSSION TITLE
onnxruntime inference example in C++

### DISCUSSION THREAD

**Discussion by 0checkmate0** [microsoft/onnxruntime#10073] [Other Q&A]
Hi, I'm wondering if there is any c++ examples to show how to do inference with a finedtuned bert model  using onnxruntime? We are interested in deploying a bert model trained with pytorch for multiclass classification task in C++ app. 
Thank you in advance

**Comment by ytaous** [comment]
hi, have u checked out the online doc and see if they might be helpful?
https://onnxruntime.ai/docs/tutorials/api-basics
https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx
